### ISSUE TITLE
Changing the current menu structure

### ISSUE THREAD

**Issue by gunchleoc** [widelands/widelands_website#82]
The current menu structure is overcrowded. This need fixing, as the page in resolutions 800x600 and lower looks awful, making it probably unusable for netbook users (can't tell myself).

Original description:
I think that this would be a good change, since the bare gets a bit crowded on resolutions lower than 1280x1024. This was tested, when the spacing was not removed. After the removal bar is still normal (1024x768), but in resolutions like 800x600 it gets wrapped. The only way to remove this effect, would be making less links.

So back to topic. The Screenshots link should be (IMHO) incorporated into Downloads menu, and Downloads text changed to 'Media'. This is just a suggestion. Maybe a blueprint for making the menu more compact would be better. (OK, this was more than a bug report...)

**Comment by gunchleoc** [comment]
(by sirver)
I agree that the menu is overcrowded, but we must keep the Downloads link as first class citizen around at all costs. According to our logs it is the most important link on the main page. Let's discuss this throughoutly.

**Comment by gunchleoc** [comment]
(by fakeuserxxxx-deactivatedaccount)
That's why I opened this bug. My another suggestion would be this:
Changing the development link into a menu of this structure (what's in a bracket, is a link):
* Development (/wiki/MainPage/#development)
|  Development (/wiki/DevelopmentPage/)
|  Maillists (/wiki/MailLists/)
|  Developers (/wiki/DevelopersPage/)

Also, to make it more compact, what about moving 'News' under 'Home' and/or 'FAQ' under 'Wiki (Help & Docu.)'?

**Comment by gunchleoc** [comment]
(by sirver)
I agree moving FAQ under wiki. I also agree to add links to the mailinglists; i think a link to the Wanna Help page is more needed than one to the developers.
News archive can be moved under home, but then home makes no sense anymore and needs renaming.

**Comment by gunchleoc** [comment]
(by fakeuserxxxx-deactivatedaccount)
Well, I though at first that it wouldn't be that bad:
* Home (/)
|  Home (/)
|  News (/news/)

But then, I see how weird it looks, without using a different name. I guess that moving it wouldn't make much difference.

**Comment by gunchleoc** [comment]
(by fakeuserxxxx-deactivatedaccount)
I decided to add the links to development pages (including developers and wanna help) now, so it is at least a bit easier to move around. (r163)

**Comment by gunchleoc** [comment]
(by sirver)
Thanks. I committed that to the page.

**Comment by gunchleoc** [comment]
(by fakeuserxxxx-deactivatedaccount)
There was a suggestion on the forums (loong time ago - http://wl.widelands.org/forum/topic/93/ ) to make sub-menus appear only when the mouse is over the arrow. It's an easy thing to achieve, but I'm not sure whether it should be made. Some opinions (at least here)?

**Comment by gunchleoc** [comment]
(by sirver)
Mmh, I think that's a bit counter intuitive. Maybe we could just make the delay (before the menu slides out) longer?

**Comment by gunchleoc** [comment]
(by fakeuserxxxx-deactivatedaccount)
I moved the FAQ as said before (seems that it hasn't been done yet, so just a minor change).

I tried to find a way of increasing the delay, I suppose that there could be something in onBeforeShow to make it wait a second or two. But I couldn't find anything that's got such an usage (setTimeout is not of any use), except for a few functions that mimic it.

I'd say that somehow modifying this function ( http://coderscult.com/javascript-ajax/2009/02/05/javascript-sleep-function/ ) and then using it might be some kind of solution.
